front_end/core/react_native/generated/type_defs/ReactNativeChromeDevToolsDebuggerEventTypes.d.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * Copyright 2024 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ *
+ * @generated by WWW D54582949
```

Comment:
Ideally this should be a signedsource comment rather than a diff reference. You can put a diff reference in a text file, or just in the commit message when updating any of these files' contents. This way there would be a clear source of truth for the latest WWW diff that touches these definitions. (Especially in the case of partial updates)